src/main/java/com/microsoft/alm/gitcredentialmanager/Program.java

```diff
@@ -1188,6 +1188,11 @@ private static void enableDebugTrace()
                     ? StorageProvider.SecureOption.PREFER
                     : StorageProvider.SecureOption.MUST;
             final com.microsoft.alm.storage.SecretStore<Token> tokenSecretStore = StorageProvider.getTokenStorage(true, secureOption);
+            if (tokenSecretStore == null) {
+                throw new RuntimeException("Cannot access OS secure storage. " +
+                        "You may opt-in to store credentials in insecure storage by running " +

```

Comment:
To add information about what we mean by insecure, consider changing to:
`throw new RuntimeException("Secure credential storage is not available on this operating system. " +`
`"You may opt-in to store credentials in an unencrypted file under your user home directory by running " +`